version: '2'
services:
  geth:
    image: daocloud.io/alphabeta_com/geth
    restart: on-failure
    environment:
      - WS_SERVER ws://blockchain.daocloud.io:3000
      - WS_SECRET daocloud-eth-net-stats-secret
      - NODES=enode://4356fb340716c7efed5c35dedcd89c0f42af5b974ababd9c4d60a0df6a949500258a045e4286100a6276cc339aacab4de44576c68659736b2d2df449f7dc1759@47.90.1.168:30303
    volumes:
      - /etc/localtime:/etc/localtime:ro
    ports:
      - "30303:30303"
      - "30303:30303/udp"
      - "8545:8545"
  server:
    image: daocloud.io/alphabeta_com/hub-blockchain
    restart: on-failure
    volumes:
    - /var/run/docker.sock:/var/run/docker.sock
    ports:
    - "8000:80"
